EXERCISE: VoteWizard - Poll #2 (#2381)
makerdao/community: governance-exercises/VoteWizard_poll_exercise_02/Derecognition of Clive AD Due to Compromised Operational Security - July 28, 2023.md

---

> title: Derecognition of Clive AD Due to Compromised Operational Security - JULY 28, 2023

Reviewer: Caps for date? A choice or unintentional? Assuming you're applying it consistently, works fine.

> summary: Signal your support or opposition to derecognition of Clive AD due to compromised operational security. A community member, Diane, posted materials claiming that they could reveal the delegate's identity.

Reviewer: Should remove second sentence here. The summary should still be primarily talking about the vote. Maybe the below:
```suggestion
summary: Signal your support or opposition to derecognition of Clive AD due to potentially compromised operational security, as reported publicly by Diane.
```

Really not convinced you need to mention Diane at all though.

discussion_link: https://forum.makerdao.com/t/...
parameters:
    input_format: single-choice
    victory_conditions:
        - { type : plurality }
    result_display: single-vote-breakdown
version: v2.0.0
options:
   0: Abstain
   1: Yes
   2: No

> start_date: 2023-07-28T16:00:00

Reviewer: July 28 isn't a Monday. Assuming you were looking at August here.

Reviewer: Coming back to the whole self-review thing :)

end_date: 2023-07-31T16:00:00
---
# Poll: Derecognition of Clive AD due to compromised operational security - JULY 28, 2023

The Governance Facilitators have placed a Governance Poll into the voting system. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) has been posted as per [MIP106c2.6.6.2.1A](https://mips.makerdao.com/mips/details/MIP106#6-6-current-incubating-ecosystem-actors) and will be active for three days beginning on Monday, July 28 at 16:00 UTC.

**This is a binary vote.**
- **You may vote for a single option.**
- **You should vote for the option which you prefer.**
- **If you would accept either option, you should vote 'Abstain'.**

## Review

> The community can vote in this poll to express support or opposition to the potential derecognition of Clive AD due to compromised operational security.

Reviewer: It's worth being kind of clear around what's happening here. community is voting on whether these materials constitute significant suspicion which leads to the Facilitator being required to offboard given MIP101.

Arguably the facilitator shouldn't be polling on this at all, but given the exercise, it was assumed that they were doing so.

Reviewer: Probably don't want 'potential' in there either.

Materials posted by a community member, Diane, claim that they could reveal the delegate's identity. As per the [Atlas](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-privacy), if these materials constitute significant suspicion, then the aligned delegate has to be derecognized.

Please review the discussion [thread](https://forum.makerdao.com/t/...) and the materials provided by Diane in [this post](https://forum.makerdao.com/t/...) to help inform your position before voting.

## Outcomes

**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
* The Aligned Delegate will be derecognized.

> * Half of the Aligned Delegate Buffer will be confiscated and used as the whistleblower bounty.

Reviewer: Arguably this is a separate concern, and should be dealt with by the facilitator without being constricted by the result of the vote. 

Reason for this is that Diane might have been a complete dick about the whole thing, to the point where voters want to disincentivise that behaviour, but also see the need to get rid of Clive.
